<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QToolBox" name="toolBox">
      <widget class="QWidget" name="page_1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>781</width>
        <height>371</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Archives222</string>
      </attribute>
      <layout class="QGridLayout" name="archiveLayout">
      </layout>
      </widget>
      <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>781</width>
        <height>371</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Prune Options and Archive Naming</string>
      </attribute>
      <layout class="QGridLayout" name="pruneLayout">
      </layout>
      </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
